services/auth/src/main/java/org/eclipse/hono/service/auth/impl/AuthenticationEndpoint.java

```diff
@@ -83,6 +83,11 @@ public final void onLinkAttach(final ProtonConnection con, final ProtonReceiver
         super.onLinkAttach(con, receiver, targetResource);
     }
 
+    @Override
+    public void onConnectionClosed(final ProtonConnection connection) {
+        // we have nothing to clean up
```

Comment:
IMHO this should go into a JavaDoc comment

Reply:
None of the other public methods have Javadoc … I just tried to blend in :grin: … I'll add something.